Segment/Internal/SEGIntegrationsManager.m

```diff
@@ -485,9 +496,9 @@ - (void)refreshSettings
                         NSMutableDictionary *newSettings = [self.configuration.defaultSettings serializableMutableDeepCopy];
                         NSMutableDictionary *integrations = newSettings[@"integrations"];
                         if (integrations != nil) {
-                            integrations[@"Segment.io"] = @{@"apiKey": self.configuration.writeKey, @"apiHost": [SEGUtils getAPIHost]};
+                            integrations[kSEGSegmentDestinationName] = [self segmentSettings];
                         } else {
-                            newSettings[@"integrations"] = @{@"integrations": @{@"apiKey": self.configuration.writeKey, @"apiHost": [SEGUtils getAPIHost]}};
```

Comment:
So this would only happen if integrations never came down via settings? ie offline mode?

Reply:
yeah exactly.